pull_request_template.md

```diff
@@ -0,0 +1,38 @@
+# Pull Request Template
+
+#issue
+
+## Description
+
+Please include a summary of the change and which issue(s) is addressed. Please also include relevant motivation and context.
+
+## How Can This Be Tested/Reviewed?
+
+Provide instructions so we can review.
+
+Describe the tests that you ran to verify your changes. Please also list any relevant details for your test configuration.
+
+## Checklist:
+
+- [ ] My code follows the style guidelines of this project
+- [ ] I have added QA notes to the issue
+- [ ] I have performed a self-review of my own code
+- [ ] I have reviewed the changes in a desktop view
+- [ ] I have reviewed the changes in a mobile view
+- [ ] I have made any corresponding changes to the documentation
+- [ ] My changes generate no new warnings
+- [ ] I have added tests that prove my fix is effective or that my feature works
+- [ ] New and existing unit tests pass locally with my changes
+- [ ] I have exported any new pieces added to ui-components
+- [ ] I have documented this change in the changelog, and any breaking changes are well described
+
```

Comment:
I'm wondering if we should add a check for storybook as well? Something like `I have added or updated the corresponding storybook if necessary`

Reply:
Added!